carmm/build/slab_consistent_bulk_generator.py

```diff
@@ -0,0 +1,106 @@
+
+# This functionality will help to generate a bulk model from a slab.
+
+from ase import Atoms
```

Comment:
Imports should be in functions where used, not at the class level.

Reply:
Will change it!!